sphinxlint/__main__.py

```diff
@@ -109,6 +119,16 @@ def __call__(self, parser, namespace, values, option_string=None):
         help="comma-separated list of fields used to sort errors by. Available "
         f"fields are: {SortField.as_supported_options()}",
     )
+    parser.add_argument(
+        "-j",
+        "--jobs",
+        metavar="N",
+        action=StoreNumJobsAction,
+        help="Run in parallle with N processes, defaults to 'auto', "
+        "which sets N to the number of logical CPUs."
+        "Values <= 1 are all considered 1.",
```

Comment:
Is this the same behaviour as sphinx-build for N <= 1?

Reply:
Don't really know, but I thought it'd be the less error prone.